examples/samplers/DEMetropolisZ_EfficiencyComparison.ipynb

```diff
@@ -4,15 +4,13 @@
    "cell_type": "markdown",
```

Comment:
<p>&gt; chain&nbsp;&nbsp;int32 0 reached the maximum tree depth. Increase <code>max_treedepth</code>, increase <code>target_accept</code> or reparameterize.&nbsp;</p><p> </p><p>This indicates non-ideal NUTS tuning, but the result that NUTS is faster &amp; more efficient still holds, so a small comment that this can be ignored would be sufficient.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/509/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>